assets/src/styles/components/_History.scss

```diff
@@ -0,0 +1,99 @@
+
+.History_Diff-Added {
+	background: #daf2d6;
```

Comment:
Should we make this a variable? Looks like it's used in a few places and can then be customised on child themes if need be